compiler/luci/service/src/CircleTypeInferenceRule.cpp

```diff
@@ -50,7 +50,14 @@ struct TypeInferenceAlgorithm final : public luci::CircleNodeVisitor<loco::DataT
     return loco::dtype_get(node->input());
   }
 
-  loco::DataType visit(const luci::CircleCast *node) final { return node->out_data_type(); }
+  loco::DataType visit(const luci::CircleCast *node) final
```

Comment:
I think that we can go with following code.
```cpp
  loco::DataType visit(const luci::CircleCast *node) final
  {
    return node->dtype();
  }
```

In `compiler/luci/import/src/Nodes/CircleCast.cpp`, it is already checked that `node->out_data_type() == node->dtype()` if `CircleCast` has option.